### PR TITLE
workingBranch

### PR DESCRIPTION
**Pull request by AlixWagner**
Updated the html in the "Results" section to allow room for the JS pushes to page & added some divs for styling purposes (ease of moving items when transitioning from desktop to mobile & for use of position absolute/relative)

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -9,47 +9,73 @@
...
-                    <option value="tequila">Tequila</option>
-                    <option value="triplesec">Triple Sec</option>
-                    <option value="vodka">Vodka</option>
-                    <option value="whiskey">Whiskey</option>
-                </select>
-            </form>
+            <div class="wrapper">
+                <!-- Can change the colours of the cocktail image depending on our colour palette for teh app: -->
+                <img src="./styles/assets/cocktailImage.svg"
+                    alt="A vector image of a cute sparkly orange cocktail in a soft blue glass" />
```
Comment (rachellehanna) [inline index.html:15]: I love your image!

File: index.html [hunk 2]
```diff
@@ -9,47 +9,73 @@
...
+            </section>
+            <!-- .recipe section will be revealed when .reveal button is pressed: -->
+            <section class="recipe">
+                <div class="drinkImage">
+                    <!-- strImageSource will be appended here (.appendChild) -->
+                </div>
+                <!-- .recipeContainer div will just be for styling purposes: -->
+                <div class="recipeContainer">
+                    <div class="ingredients">
+                        <h4>INGREDIENTS</h4>
```
Comment (rachellehanna) [inline index.html:66]: Should we use `text-transform` instead of caps in the HTML?

**Review by rachellehanna — CHANGES_REQUESTED** [review]

File: index.html [hunk 3]
```diff
@@ -9,47 +9,73 @@
...
-                    <option value="vodka">Vodka</option>
-                    <option value="whiskey">Whiskey</option>
-                </select>
-            </form>
+            <div class="wrapper">
+                <!-- Can change the colours of the cocktail image depending on our colour palette for teh app: -->
+                <img src="./styles/assets/cocktailImage.svg"
+                    alt="A vector image of a cute sparkly orange cocktail in a soft blue glass" />
+                    <img src="https://images.unsplash.com/photo-1611873027887-19a588803816?ixlib=rb-1.2.1&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=627&q=80"
+                        alt="#" />
```
[the remark this answers is not in the document]
Comment (AlixWagner) [reply index.html:17]: Sure - just didn't want to until confirmed!

File: index.html (hunk 1 above)
Comment (AlixWagner) [reply index.html:15]: Sweet! We can mess around with the colours later but thought it would be fun!

File: index.html (hunk 2 above)
Comment (AlixWagner) [reply index.html:66]: Yes. Absolutely.

**Pull request merged**